mtgjson5/classes/mtgjson_identifiers.py

```diff
@@ -24,11 +24,13 @@ class MtgjsonIdentifiersObject:
     scryfall_oracle_id: Optional[str]
     tcgplayer_product_id: Optional[str]
     mtgjson_v4_id: Optional[str]
+    tcgplayer_sku_ids: List[Dict[str, str]]
```

Comment:
Yeah we're gonna have to address this... @kodabb maybe you have some suggestions?

Reply:
I'd honestly move these details to a separate dictionary, so that you don't pollute `identifiers` with too much information and can have any format for the new entry

I would propose `tcgplayer_sku[string]{language, condition}`

example

    "tcgplayer_skus" : {
        "3455" : {
            "product_id": "1348"
            "language": "EN",
            "condition": "LP".
            ...
        }
    }